src/main/java/be/seeseemelk/mockbukkit/ServerMock.java

```diff
@@ -450,6 +450,11 @@ public void addWorld(WorldMock world)
 		worlds.add(world);
 	}
 
+	public boolean removeWorld(WorldMock world) {
```

Comment:
Does this need to be public?

Reply:
The addWorld method above was public, so i just followed that. I can change to private if that's your preference.